.github/workflows/main.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```

Comment:
GitHub provides Python 3.10 as version 3.10.0

Reply:
What's the difference?

Reply:
Oh...it's reading 3.10 as 3.1...ooof